code/_onclick/click.dm

```diff
@@ -327,15 +327,18 @@
 		ML.pulled(src)
 
 /mob/living/CtrlClick(mob/user)
-	if(isliving(user) && Adjacent(user) && !user.incapacitated())
-		if(world.time < user.next_move)
-			return FALSE
-		var/mob/living/user_living = user
-		if(user_living.apply_martial_art(src, null, is_grab=TRUE) == MARTIAL_ATTACK_SUCCESS)
-			user_living.changeNext_move(CLICK_CD_MELEE)
-			return
-	else
-		..()
+	if(!isliving(user) || !Adjacent(user) || user.incapacitated())
+		return ..()
+
+	if(world.time < user.next_move)
+		return ..()
+
+	var/mob/living/user_living = user
+	if(user_living.apply_martial_art(src, null, is_grab=TRUE) == MARTIAL_ATTACK_SUCCESS)
+		user_living.changeNext_move(CLICK_CD_MELEE)
```

Comment:
huh, I didn't notice this before, but it looks like martial art grabs use CLICK_CD_MELEE instead of CLICK_CD_GRAB. weird, but I guess that changing that could cause some balance issues.

Reply:
There is no define called CLICK_CD_GRAB in the code.

Reply:
my bad, it's named "CLICK_CD_GRABBING", not "CLICK_CD_GRAB"

:agony:

Reply:
Not a code review.

Reply:
It became CLICK_CD_MELEE in #57602 yesterday, perhaps this was simply incorrect? @necromanceranne